easybuild/easyblocks/a/amber.py

```diff
@@ -121,13 +121,24 @@ def install_step(self):
 
         # define environment variables for MPI, BLAS/LAPACK & dependencies
         mklroot = get_software_root('imkl')
+        openblasroot = get_software_root('OpenBLAS')
         if mklroot:
             env.setvar('MKL_HOME', mklroot)
+        elif openblasroot:
+	    lapack = os.getenv('LIBLAPACK')
+	    if lapack is None:
+		raise EasyBuildError("LIBLAPACK (from OpenBLAS) not found in environement. Not building with a lapack capable toolchain?")
```

Comment:
@akesandgren This line is too long (> 120 chars). I would just drop the 2nd sentence, since a situation where `OpenBLAS` is there but `$LIBLAPACK` is not defined is quite strange...